tools/slims.py

```diff
@@ -171,7 +172,7 @@ def get_pair_dict(Sctx, run_tag, logger):
         pair.slims_info = translate_slims_info(pair)
         pair_dict[pair.slims_info["content_id"]] = [pair.slims_info["tumorNormalType"], pair.slims_info["tumorNormalID"]]
```

Comment:
The return from `translate_slims_info()` is a dict, and from what I can see there is no point in adding that as an attribute to the `record` class returned from the `slims_connection.fetch()` call. Store the translated slims info dict outside `pair` and call on that for information.

As it stands now it looks like the information will be used later (as in further downstream) via `pair.slims_info` but I can't see it.

Reply:
Solved in commit @ d42cd8f
Removed that it uses translate_slims_info, it is not needed.